microsoft/testsuites/performance/storageperf.py

```diff
@@ -329,4 +329,11 @@ def _perf_premium_datadisks(
 
     def after_case(self, log: Logger, **kwargs: Any) -> None:
         node: Node = kwargs.pop("node")
+
+        # stop nfs server if it's running
+        nfsserver = node.tools[NFSServer]
```

Comment:
will other cases which don't use nfsserver be installed the tool here?

Reply:
Nice catch. This should prompt installation. Maybe we can make this static function in the class.

Reply:
This is not applicable anymore